src/nupic/algorithms/backtracking_tm.py

```diff
@@ -1355,7 +1357,7 @@ def getSegmentOnCell(self, c, i, segIdx):
     return retlist
 
 
-  class _SegmentUpdate(Serializable):
+  class _SegmentUpdate(object):
```

Comment:
Segment and SegmentUpdate are not intended to be used outside BacktrackingTM - they are implementation details. So we don't need the common serialization interface for them. Additionally, they require additional information for deserialization, so the interface is necessarily different. Do you have any suggested changes or just curious?